core/rfb.js

```diff
@@ -337,6 +337,16 @@ export default class RFB extends EventTargetMixin {
         }
     }
 
+    get useNativeDpi() { return this._nativeDpi; }
+    set useNativeDpi(nativeDpi) {
```

Comment:
Should this be `useNativeDPI`? What does JavaScript APIs usually do with abbreviations?